client/src/main/java/tds/config/ClientSystemFlag.java

```diff
@@ -61,6 +81,9 @@ public void setDescription(String description) {
         this.description = description;
     }
 
+    /**
+     * @return The most recent date and time when the {@link ClientSystemFlag} was changed.
+     */
     public Timestamp getDateChanged() {
```

Comment:
I started this refactor awhile ago, so it's out of date w/some of the more recent decisions we've made.  I'll go back and update to match our current standards.